library/src/main/java/org/apache/apex/malhar/lib/utils/IOUtils.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.apex.malhar.lib.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class IOUtils
+{
+  /**
+   * Utility method to copy partial data from input stream to output stream.
+   * @param inputStream        input stream
+   * @param inputStreamOffset  offset in the input stream till which the data is copied.
+   * @param outputStream       output stream
+   * @throws IOException
+   */
+  public static void copyPartial(InputStream inputStream, long inputStreamOffset, OutputStream outputStream)
```

Comment:
IOUtils.copyLarge will copy all the data from input to output stream.
Needed to copy data till a particular offset from the input to output. This is common in AbstractFileOutput operator which can use this util function.